modules/freewheel-sspBidAdapter.js

```diff
@@ -76,7 +76,7 @@ function getPricing(xmlNode) {
     var priceNode = pricingExtNode.querySelector('Price');
     princingData = {
       currency: priceNode.getAttribute('currency'),
-      price: priceNode.textContent || priceNode.innerText
```

Comment:
You can remove freewhell from this pr because #11534

Reply:
done